test/config_parser_SUITE.erl

```diff
@@ -1006,6 +1296,8 @@ handle_config_option(#local_config{key = K1, value = V1},
 handle_config_option(Opt1, Opt2) ->
     ?eq(Opt1, Opt2).
 
+compare_values(_, X, X) -> %% I am a simple man
+    ok;
```

Comment:
Already handled by the last clause, remove.

Reply:
no, it fails for some option.